examples/redux/README.md

```diff
@@ -0,0 +1,41 @@
+# Usage with Redux
+
+#### 1. Add dependencies
+``` bash
+npm install airbrake-js redux-airbrake --save
+```
+
+#### 2. Import dependency
+``` js
+import airbrakeMiddleware from 'redux-airbrake';
+```
+
+#### 3. Configure & add middleware
+``` js
+const errorMiddleware = airbrakeMiddleware({
+    projectId: '******',
+    projectKey: '**************'
+});
```

Comment:
Could you change this to accept airbrake-js Client? This way
- it is consistent with other middlewares
- user can configure client by adding filters/additional info